website/content/partials/alerts/experimental.mdx

```diff
@@ -1,7 +1,6 @@
-<Note title="Experimental feature">
+<Warning title="Experimental">
 
-  Experimental features are features in an alpha state or available early in development to
-  provide as a tech demo experience and are subject to change.
-  **We strongly discourage using experimental features in production deployments of Vault.**
+  Experimental features are tested but unproved. Until the feature is verified
+  through heavy production use, proceed with caution.
 
-</Note>
\ No newline at end of file
+</Warning>
```

Comment:
By pure chance someone created a file with this exact same name yesterday before me. I changed the tags in my PR to alpha instead now because this description of experimental does not sound like it fits well with what we're aiming to provide here.

Reply:
That was me, sorry about that! I agree that "alpha" is a better fit 👍🏽